### ISSUE TITLE
Deleted network nodes still analyzed and seen as orphan nodes

### ISSUE THREAD

**Issue by A67-A67** [vmarc/knooppuntnet#47]
The network nodes below are not network nodes anymore (rwn_ref is removed), but they are still analyzed. Therefore they are incorrectly seen as orphan nodes:
- [https://knooppuntnet.nl/nl/node/6234320214](https://knooppuntnet.nl/nl/node/6234320214)
- [https://knooppuntnet.nl/nl/node/6234320386](https://knooppuntnet.nl/nl/node/6234320386)

**Comment by vmarc** [comment]
@A67-A67 Thanks for reporting this problem.

Replication file 003/338/166 contains the deletions for both nodes:

```
  <delete>
    <node id="6234320214" version="2" timestamp="2019-01-25T15:18:24Z" uid="294028" user="dvdhoven" changeset="66634908" lat="53.3538907" lon="6.3366934"/>
    <node id="6234320386" version="2" timestamp="2019-01-25T15:18:24Z" uid="294028" user="dvdhoven" changeset="66634908" lat="53.3539175" lon="6.3366895"/>
  </delete>
```

The analysis logic did correctly recognize these deletions, as shown in the history in the node pages [6234320214](https://knooppuntnet.nl/en/node/6234320214) en [6234320386](https://knooppuntnet.nl/en/node/6234320386) facts: _WasOrphan_ and _Deleted_.

But the node was not set to non-active!

The  [code](https://github.com/vmarc/knooppuntnet/blob/master/production/core/src/main/scala/kpn/core/engine/changes/orphan/node/OrphanNodeDeleteProcessorImpl.scala) arround line 36 should set the node to non-active _(active=false)_, but somehow this does not work ok.

Has to be further investigated.

**Comment by vmarc** [comment]
The 2 nodes have been manually set to non-active in the database. This way they are no longer reported as orphan nodes.

This however does not resolve the actual problem yet: the nodes should have been set to non-active automatically by the analysis logic.

**Comment by vmarc** [comment]
Resolved in release v4.0.
